verilog/j1eforth/j1eforth-plus/README.md

```diff
@@ -0,0 +1,99 @@
+eForth for the J1 Simulator and actual J1 FPGAs
```

Comment:
If we convert this file into an RST document ([pandoc](https://pandoc.org/) can do it automagically) then it can be used as a start for being included in the published documentation.

Reply:
If any of the READMEs in the subdirs of this repo are expected to be included in the published documentation, I would recommend writing them in rst directly. That is, use `README.rst` instead of `README.md`.

Markdown documents can be included in Sphinx directly by using extension `recommonmark`. Unfortunately, markdown is poorly specified for documents composed by multiple files, so x-ref features are non-existent. Therefore, I think it is undesirable to do so. Using pandoc provides a similar result. In any case, the markdown document would be included in the docs, but it would feel isolated.